build.gradle

```diff
@@ -178,24 +177,9 @@ processResources {
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
-def String deriveVersion(){
-    def stdout = new ByteArrayOutputStream()
-    try {
-        logger.info("path is $System.env.PATH")
-        exec {
-            commandLine "git", "describe", "--always"
-            standardOutput = stdout;
+final isRelease = Boolean.getBoolean("release")
+version = isRelease ? gitVersion() : gitVersion() + "-SNAPSHOT"
```

Comment:
Will this have the "dirty" marker on it even when no tracked files have been changed?